geany.appdata.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<component>
+  <id type="desktop">geany.desktop</id>
```

Comment:
Yeah I wasn't clear: I meant, shouldn't the appdata file name be `geany.desktop.appdata.xml`, following this `ID`?  The linked doc isn't clear on that, but uses the `%{id}` placeholder there